study-builder/studies/testboston/emails/adhoc_completed.en.html

```diff
@@ -0,0 +1,223 @@
+                              </td>
+                            </tr>
+                            </tbody></table>
+
+                            <table class="module" role="module" data-type="text" border="0" cellpadding="0" cellspacing="0" width="100%" style="table-layout: fixed;" data-muid="13fe2859-7e54-40df-8fd6-536a6cf6edbb" data-mc-module-version="2019-10-22">
+                              <tbody>
+                              <tr>
+                                <td style="padding:20px 20px 20px 20px; line-height:24px; text-align:inherit; background-color:#ffffff;" height="100%" valign="top" bgcolor="#ffffff" role="module-content"><div><div style="font-family: inherit; text-align: inherit">Dear Study Staff,</div>
+                                  <div style="font-family: inherit; text-align: inherit; color: #464f61;"><br></div>
+                                  <div style="font-family: inherit; text-align: inherit; color: #464f61;">TestBoston participant with hruid <strong>{{participant_hruid}}</strong> and invite code <strong>{{invitationId}}</strong> has submitted their Ad Hoc Symptom survey.</div>
+                                  <div style="font-family: inherit; text-align: inherit; color: #464f61;"></div><div></div></div></td>
+                              </tr>
+                              </tbody>
+                            </table>
+
+                            <table border="0" cellpadding="0" cellspacing="0" class="module" data-role="module-button" data-type="button" role="module" style="table-layout:fixed;" width="100%" data-muid="94a7b047-b437-4626-b5ad-1f2dc3879109">
+                              <tbody>
+                              <tr>
+                                <td align="left" bgcolor="" class="outer-td" style="padding:0px 20px 0px 20px;">
+                                  <table border="0" cellpadding="0" cellspacing="0" class="wrapper-mobile" style="text-align:center;">
+                                    <tbody>
+                                    <tr>
+                                      <td align="center" bgcolor="#0967d2" class="inner-td" style="border-radius:6px; font-size:16px; text-align:left; background-color:inherit;">
+                                        <a style="background-color:#0967d2; border:0px solid #165396; border-color:#165396; border-radius:8px; border-width:0px; color:#ffffff; display:inline-block; font-size:16px; font-weight:normal; letter-spacing:0px; line-height:normal; padding:16px 28px 16px 28px; text-align:center; text-decoration:none; border-style:solid;" target="_blank"  href="{{baseWebUrl}}/prism/activity-link?invitationId={{invitationId}}&instanceGuid={{activityInstanceGuid}}">Click here to view in Prism Dashboard</a>
```

Comment:
The `/prism/activity-link` URL will bring study staff directly to the submitted adhoc symptom survey. See https://github.com/broadinstitute/ddp-angular/pull/370.